template.yaml

```diff
@@ -621,3 +621,59 @@ Resources:
       ManagedPolicyArns:
         - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWSBackupServiceRolePolicyForBackup"
         - !Sub "arn:${AWS::Partition}:iam::aws:policy/AWSBackupServiceRolePolicyForS3Backup"
+
+  LambdaVersionCleanupFunctionAccessLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub '/aws/lambda/${LambdaVersionCleanupFunction}'
+      RetentionInDays: 90
+
+  # Lambda function to delete old versions
+  LambdaVersionCleanupFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: delete_old_lambda_versions.lambda_handler
+      Runtime: python3.11
+      Timeout: 900
+      MemorySize: 128
+      Role: !GetAtt LambdaVersionCleanupRole.Arn
+      CodeUri: .
+      Events:
+          # Optional: Schedule to trigger the Lambda function
+          ScheduledEvent:
+            Type: Schedule
+            Properties:
+              Schedule: rate(7 days)  # Run every 7 days
+              Name: CleanupLambdaVersionSchedule
+
+  # IAM Role for the Lambda function
```

Comment:
Its better to add comments as "Description:" That way we can also view them in AWS

Reply:
Should also specify which lambda function in the comment

Reply:
Resolved